src/deluge/gui/ui/keyboard/chords.h

```diff
@@ -94,6 +121,30 @@ extern const Chord k13;
 extern const Chord kM13;
 extern const Chord kMinor13;
 
+const std::array<Chord, kDisplayHeight> majorChords = {
```

Comment:
When it did, the release build right. Like all of the arrays (or vectors, because I originally wanted them as vectors) elements were null, so it got some funky behavior. You can see that behavior with trapper's message here (https://discord.com/channels/608916579421257728/1262843243846959225/1277405330334027919). In the debug build it worked fine split across though. 

Though the bin size for my local release build is like 11 kb bigger with it defined in the .h file, so it doesn't seem to be that big of a problem? With the vectors it was like 30 kb bigger I think in the .h, which is why I went with the arrays, even if it meant a little more work in `chord_keyboard.cpp`. Though if you can figure out how to get it be split, that would be cool?

Reply:
Yeah the potential problem is that every file that imports this header will create a new copy of the arrays, and what we'd want is all uses to refer to the same underlying data 


Being all zeroes sounds like an initialization bug. Vectors need dynamic memory at runtime which might not be set up before they're initialized

Reply:
It occurred when I swapped them out for arrays too. Want me to push a version with arrays split up so you can check it out?

Reply:
yeah sure

Reply:
I split the definition up (as arrays still) and it only plays the root in a column, and the display sometimes produces a random extra symbol. Not sure why. Maybe you can see something I am doing wrong?

Reply:
Pushed a fix, it's from placing the extern variables in the array instead of referring to them. Arrays own their storage so that means the array got its own copy of the variables which didn't get initialized by chords.cpp. 

I just changed that array to hold pointers to the extern const chord lists. This lowers the bin size by avoiding duplication and also ensures that the array holds the correct data